html/elements/iframe.json

```diff
@@ -108,6 +108,1327 @@
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror"
+              },
+              "status": {
+                "experimental": true,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "attribution-reporting": {
+            "__compat": {
+              "spec_url": "https://wicg.github.io/attribution-reporting-api/#permission-policy-integration",
+              "support": {
+                "chrome": {
+                  "version_added": "117"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror"
+              },
+              "status": {
+                "experimental": true,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "autoplay": {
+            "__compat": {
+              "spec_url": "https://html.spec.whatwg.org/multipage/infrastructure.html#autoplay-feature",
+              "support": {
+                "chrome": {
+                  "version_added": "64"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "74"
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror"
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "battery": {
```

Comment:
Looks like adding `battery` is not a good idea after all, because it isn't supported in any browser, so the linter complains:

```
browsers dating back two or more years ago.
 ✖ html.elements.iframe.allow.battery - Error → feature was never implemented.
```